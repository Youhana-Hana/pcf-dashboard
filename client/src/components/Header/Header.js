import React, { Component } from 'react';
import {
  NavbarBrand,
} from 'reactstrap';

import logo from '../../../public/img/biclogo.jpg';
import boschLogo from '../../../public/img/boschlogo.jpg';

class Header extends Component {
  render() {
    return (
      <header className="app-header" >
            <div className="header">
            <div className="col_1"><img src={logo}/></div>
            <div className="col_2 title">Bosch IoT Cloud</div>
            <div className="col_3 title">PCF Automation</div>
            <div className="col_4"><img src={boschLogo} /></div>
            </div>
      </header>
    )
  }
}

export default Header;
