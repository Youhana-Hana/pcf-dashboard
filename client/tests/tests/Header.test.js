import React from 'react';
import renderer from 'react-test-renderer';
import Header  from '../../src/components/Header/';

it('test to see if  renders correctly', () => {
  const header = renderer
  .create(<Header />)
  .toJSON();
  expect(header).toMatchSnapshot();
});
