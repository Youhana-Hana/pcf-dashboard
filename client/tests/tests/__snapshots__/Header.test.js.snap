// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test to see if  renders correctly 1`] = `
<header
  className="app-header"
>
  <div
    className="header"
  >
    <div
      className="col_1"
    >
      <img
        src="test-file-stub"
      />
    </div>
    <div
      className="col_2 title"
    >
      Bosch IoT Cloud
    </div>
    <div
      className="col_3 title"
    >
      PCF Automation
    </div>
    <div
      className="col_4"
    >
      <img
        src="test-file-stub"
      />
    </div>
  </div>
</header>
`;
